src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilsTest.java

```diff
@@ -63,7 +67,7 @@ public void testGetIntFromInaccessibleField() throws NoSuchFieldException {
 
     @Test
     public void testTokenValueIncorrect() {
-        final Integer id = Integer.MAX_VALUE - 1;
+        final Integer id = 184;
```

Comment:
I'm not sure, I got what you asked me to do... However, I got rid of `184` hardcode